provider/virt_storage/storage_volume.py

```diff
@@ -157,8 +157,11 @@ def refresh_format_by_params(self, params):
 
             backing = params.get("backing")
             if backing:
-                backing_node = "drive_%s" % backing
-                self.format.set_param("backing", backing_node)
+                if params.get("backing_null", "no") == "no":
```

Comment:
It's better to use another commit for the changes in provider

Reply:
But for this patch, it's up to you, I am also OK. For the patches pushed in future, please split it into two commits in one patch :-D

Reply:
Have splited the commit, thanks @zhencliu